src/mutation.cpp

```diff
@@ -499,23 +494,20 @@ void player::activate_mutation( const trait_id &mut )
         blossoms();
         tdata.powered = false;
         return;
-    } else if (mut == "VINES3"){
-        item newit( "vine_30", calendar::turn );
-        if ( !can_pickVolume( newit ) ) { //Accounts for result_mult
-            add_msg_if_player(_("You detach a vine but don't have room to carry it, so you drop it."));
-            g->m.add_item_or_charges(pos(), newit);
-        } else if ( !can_pickWeight( newit, !get_option<bool>( "DANGEROUS_PICKUPS" ) ) ) {
-            add_msg_if_player(_("Your freshly-detached vine is too heavy to carry, so you drop it."));
-            g->m.add_item_or_charges(pos(), newit);
-        } else {
-            inv.assign_empty_invlet(newit);
-            newit = i_add(newit);
-            add_msg_if_player(m_info, "%c - %s", newit.invlet == 0 ? ' ' : newit.invlet, newit.tname().c_str());
-        }
+    }  else if( mut == trait_SELFAWARE ) {
+        print_health();
         tdata.powered = false;
         return;
-    } else if( mut == trait_SELFAWARE ) {
-        print_health();
+    } else if ( !mdata.spawn_item.empty() ) {
+        item tmpitem( mdata.spawn_item );
+        i_add_or_drop( tmpitem );
+        if (mut == "WEB_ROPE") {
+            add_msg_if_player( _("You spin a rope from your silk.") );
+        } else if (mut == "VINES3") {
+            add_msg_if_player(_("You detach a vine from your body."));
+        } else {
+            add_msg_if_player(_("You create a %s."), tmpitem.display_name().c_str());
```

Comment:
All `add_msg_if_player` here and above should have brackets with spaces. Like `( some text here )`.